contact.js

```diff
@@ -2,6 +2,21 @@ const myForm = document.getElementById('form1');
 const email = document.getElementById('email');
 const errorMessage = document.querySelector('.email-error');
 const submitBtn = document.querySelector('.submit');
+const fullName = document.querySelector('#name');
+const textArea = document.querySelector('#text-area');
+const data = {};
+
+const getLocalData = () => {
+  let myData;
+  if (localStorage.getItem('data')) {
+    myData = JSON.parse(localStorage.getItem('data'));
+    fullName.value = myData.name;
+    email.value = myData.email;
+    textArea.value = myData.message;
+  }
+};
+
+fullName.addEventListener('click', getLocalData);
```

Comment:
- Great job for populating the local storage data to the form,👍👍 I think it would be better to populate the data when the page loaded instead of a single event listener 👌, check below reference 👇👇
```
window.onload = () => {
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  }
};
```